src/status_im/subs/contact.cljs

```diff
@@ -242,7 +309,16 @@
     (re-frame/subscribe [:contacts/contacts])
     (re-frame/subscribe [:profile/profile])])
  (fn [[{:keys [contacts admins]} all-contacts current-multiaccount]]
-   (contact.db/get-all-contacts-in-group-chat contacts admins all-contacts current-multiaccount)))
+   (get-all-contacts-in-group-chat contacts admins all-contacts current-multiaccount)))
+
+(defn- contact-by-address
+  [[addr contact] address]
+  (when (address/address= addr address)
```

Comment:
is it common to add a suffix like that in clojure?

Reply:
i've never seen this before, please ignore this namespace it's very old code